CodeEditModules/Modules/CodeFile/src/CodeEditorView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CodeFileView.swift
+//  CodeEdit
+//
+//  Created by Marco Carnevali on 17/03/22.
+//
+
+import CodeEditor
+import Foundation
+import SwiftUI
+
+/// CodeEditor is just a wrapper of the `CodeViewer` dependency
+public struct CodeFileView: View {
```

Comment:
The name of the file does not match the struct name

Reply:
good catch thank you!